### ISSUE TITLE
Enable build with Java 15

### ISSUE THREAD

**Issue by joschi** [dropwizard/metrics#1734]
There are currently some problems building Metrics 4.x with Java 15:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:testCompile (default-testCompile) on project metrics-jvm: Compilation failure
Error:  /home/runner/work/metrics/metrics/metrics-jvm/src/test/java/com/codahale/metrics/jvm/FileDescriptorRatioGaugeTest.java:[16,78] <anonymous com.codahale.metrics.jvm.FileDescriptorRatioGaugeTest$1> is not abstract and does not override abstract method getCpuLoad() in com.sun.management.OperatingSystemMXBean
```

See #1729, 10110e7daf5bd6315e012d6963c66fb1581322f8 and 4527d40573077b3ab5cfea907cb7bc4317a57d56.

**Comment by arteam** [comment]
Fixed by #1738

**Comment by arteam** [comment]
Closed by #1738
